yb-voyager/src/callhome/diagnostics.go

```diff
@@ -92,8 +92,9 @@ type TargetDBDetails struct {
 }
 
 type UnsupportedFeature struct {
-	FeatureName string `json:"FeatureName"`
-	ObjectCount int    `json:"ObjectCount"`
+	FeatureName      string `json:"FeatureName"`
+	ObjectCount      int    `json:"ObjectCount"`
+	TotalOccurrences int    `json:"TotalOccurrences"`
```

Comment:
should we omitEmpty? 
Since we are using the same struct here for other assessment issues as well, shouldn't we fill in this value of TotalOccurences there as well?